apps/worker/src/batch_announcer/batch.announcer.service.ts

```diff
@@ -0,0 +1,63 @@
+import { InjectRedis } from '@liaoliaots/nestjs-redis';
+import { Processor, WorkerHost, OnWorkerEvent, InjectQueue } from '@nestjs/bullmq';
+import { Injectable, Logger, OnApplicationBootstrap, OnModuleDestroy } from '@nestjs/common';
+import { Job, Queue } from 'bullmq';
+import Redis from 'ioredis';
+import { SchedulerRegistry } from '@nestjs/schedule';
+import { EventEmitter2 } from '@nestjs/event-emitter';
+import { ConfigService } from '../../../api/src/config/config.service';
+import { IPFSAnnouncer } from './ipfs.announcer';
+import { CAPACITY_EPOCH_TIMEOUT_NAME } from '../../../../libs/common/src/constants';
+import { IBatchAnnouncerJobData } from '../interfaces/batch-announcer.job.interface';
+
+@Injectable()
+@Processor('batchAnnouncerQueue', {
```

Comment:
same

Reply:
fixed in this https://github.com/AmplicaLabs/content-publishing-service/pull/26